hnn_core/network.py

```diff
@@ -177,39 +167,36 @@ def __init__(self, params):
         self.n_src = 0
         self.n_of_type = {}  # numbers of sources
         self.n_cells = 0  # init self.n_cells
-
-        # params of common external feeds inputs in p_common
-        # Global number of external inputs ... automatic counting
-        # makes more sense
-        # p_unique represent ext inputs that are going to go to each cell
-        self.p_common, self.p_unique = create_pext(self.params,
-                                                   self.params['tstop'])
-        self.n_common_feeds = len(self.p_common)
-        # Source list of names
-        # in particular order (cells, common, names of unique inputs)
-        self.src_list_new = self._create_src_list()
+        # Source list of names, first real ones only!
+        self.cellname_list = [
+            'L2_basket',
+            'L2_pyramidal',
+            'L5_basket',
+            'L5_pyramidal',
+        ]
```

Comment:
this attribute should be documented

Reply:
Added, but several others also remain to be documented. Leaving for a future cleanup-pr